src/util.cc

```diff
@@ -449,6 +449,70 @@ void Win32Fatal(const char* function, const char* hint) {
     Fatal("%s: %s", function, GetLastErrorString().c_str());
   }
 }
+
+/// In tests, codepage can be manually specified to override the ANSI codepage.
+bool TryConvertUtf8ToAnsi(const char* utf8, string& ansi,
```

Comment:
Base your PR off of this PR: https://github.com/ninja-build/ninja/pull/1671

Reply:
Removed UTF-8 to ANSI conversion.